code/modules/projectiles/ammunition/energy/laser.dm

```diff
@@ -2,7 +2,10 @@
 	projectile_type = /obj/projectile/beam/laser
 	e_cost = 83
 	select_name = "kill"
-
+/obj/item/ammo_casing/energy/laser/revolver
+	projectile_type = /obj/projectile/beam/laser
+	e_cost = 100
+	select_name = "kill"
```

Comment:
```suggestion
```

Reply:
Removing these takes the values from `/obj/item/ammo_casing/energy/laser` instead, which is what you want here.
Also this ought to be modularised, I can handle that if you wish.